src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -168,7 +216,37 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
         userPrefs.setAddressBookFilePath(addressBookFilePath);
     }
 
-    //========== EntityListMethods ===============
+    // ========== EntityListMethods ===============
+
+    // Tried to use streams, but streams cannot throw checked exceptions
+    public ParticipantList addAllParticipants(ParticipantList currList, ReadOnlyEntityList newList)
```

Comment:
What is the use case for these methods

Reply:
initialising the ParticipantList from Storage?